app/models/jobsModel.js

```diff
@@ -54,8 +55,28 @@ async function get(filter){
     }
 }
 
+async function dequeue(){
```

Comment:
although the functionality is dequeue, but in a database model perspective, this function should be called getByMinutesFromNow and we can add on a parameter that takes the number of mins to add.